source/adapters/hip/program.cpp

```diff
@@ -92,6 +92,26 @@ urProgramCompile(ur_context_handle_t hContext, ur_program_handle_t hProgram,
   return urProgramBuild(hContext, hProgram, pOptions);
 }
 
+UR_APIEXPORT ur_result_t UR_APICALL urProgramCompileExp(
+    ur_context_handle_t hContext, ur_program_handle_t hProgram,
+    uint32_t numDevices, ur_device_handle_t *phDevices, const char *pOptions) {
+  std::ignore = numDevices;
+  std::ignore = phDevices;
+  return urProgramBuild(hContext, hProgram, pOptions);
```

Comment:
same

Reply:
same as above, compile calls build for hip/cuda plugins so this was trying to prevent one more function jump.

However, I have gone ahead and made these functions unsupported for hip/cuda so the calls just fallback to the standard calls in pi2ur.